webportal/webapp/dev_deploy.sh

```diff
@@ -1,6 +1,19 @@
 #/bin/bash
-ng build --aot=true
-#Copy everything over to S3 bucket
-aws s3 cp --profile sdc ./dist s3://test-sdc-webportal-hosting --recursive --metadata-directive REPLACE --cache-control max-age=86400 --acl public-read
-#Bust open the cache
-aws s3 cp --profile sdc ./dist/index.html s3://test-sdc-webportal-hosting/index.html --region us-east-1 --metadata-directive REPLACE --cache-control max-age=0 --acl public-read
+
+echo "***"
+echo "Building assets..."
+ng build --configuration=dev-private
+
+
+# Use distinct dev-private bucket so we can modify URLs, etc. per the environment
+# Copy everything over to S3 bucket
+echo "Copying to s3..."
+aws s3 cp --profile sdc ./dist s3://dev-private-sdc-webportal-hosting --recursive --only-show-errors
```

Comment:
Once we do cleanup, we'll rename this to the original bucket and get rid of the custom private bucket. While we are at it, we will also want to make everything explicitly private in our s3 bucket.